src/modules/graphics/wrap_Graphics.lua

```diff
@@ -55,6 +55,8 @@ end
 function graphics.stencil(func, action, value, keepvalues)
 	love.markDeprecated(2, "love.graphics.stencil", "function", "replaced", "love.graphics.setStencilMode or setStencilState")
 
+	action = action or "replace" -- make compatible with 11.x by giving default value --
```

Comment:
Can this be `if action == nil then action = "replace" end`? To match the code below it, and to make sure any boolean is still an error like in love 11.

Reply:
Also the comment can be removed